modules/persistent-storage-csi-vsphere-migrating-cns-vols-between-datastores.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-vsphere-migrating-cns-vols-between-datastores_{context}"]
+= Migrating CNS volumes between datastores for vSphere
+
+If you are running out of space in your current datastore, or want to move to a more performant datastore, you can migrate VMware vSphere Cloud Native Storage (CNS) volumes between datastores. This applies to both attached and detached volumes.
+
+.Limitations
+* Requires VMware vSphere 8.0.2 or later or vSphere 7.0 Update 3o or later.
+
+* Only one volume can be migrated at a time.
```

Comment:
```suggestion
* Requires VMware vSphere 8.0.2 or later. As an alternative, requires vSphere 7.0 Update 30 or later.

* You can migrate only one volume at a time.
```

- Suggested change to eliminate multiple "or" and to avoid passive voice.
- Wondering if "3o" is "30"